api/models/Campaign.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * Models a DS Campaign.
+ */
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  keyword: String,
+
+  // Properties cached from DS API.
+  title: String,
+  rb_confirmed: String,
+  rb_noun: String,
+  rb_verb: String,
+  status: String,
```

Comment:
@sergii-tkachenko This corresponds to the Campaign's `field_campaign_status` in Phoenix -- which has values `active` and `closed`.